inc/Predictors/Predictor.h

```diff
@@ -17,9 +17,9 @@
 #include "Trajectory/TrajectoryService.h"
 
 namespace PCOE {
-    class Prediction {
+	class Prediction {
     public:
-        Prediction(std::vector<ProgEvent>&& events, std::vector<DataPoint>&& trajectories)
+        Prediction(std::vector<ProgEvent> events, std::vector<DataPoint> trajectories)
```

Comment:
I think the r-value constructor should remain even if a constructor that copies the args is necessary. The constructor taking values should `std::move` them to prevent an extra copy.